prebid-mobile/pbm-api/android/adunit-android.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: page_v2
+title: AdUnit - Android
+description: AdUnit - Android
+top_nav_section: prebid-mobile
+nav_section: prebid-mobile
+sidebarType: 2
+---
+
+# AdUnit Object
+{:.no_toc}
+
+The `AdUnit` object is an abstract object that cannot be instantiated. Use the [BannerAdUnit]({{site.baseurl}}/prebid-mobile/pbm-api/android/banneradunit-android.html) or [InterstitialAdUnit]({{site.baseurl}}/prebid-mobile/pbm-api/android/interstitialadunit-android.html) object to create and configure the desired type of ad unit within your app.
+
+* TOC
+{:toc}
+
+## Object
+
+### AdUnit
+
+`AdUnit` properties and methods are inherited by `BannerAdUnit` and `InterstitialAdUnit`.
+
+**Parameters**
+
+- `configId`: String containing the Prebid Server configuration ID.
+- `adType`: `BANNER` or `INTERSITIAL`. This value will be set by the object based on which type of ad unit object you create.
+
+**Properties**
+
+- `configId`: Prebid Server configuration ID.
+- `adType`: `BANNER` or `INTERSITIAL`.
+- `periodMillis`: Integer defining the refresh time in milliseconds. Default = 0, meaning no auto refresh.
+- `keywords`: ArrayList containing keys and values.
+
+## Methods
+
+### fetchDemand
+
+Trigger a call to Prebid Server to retrieve demand for this Prebid Mobile ad unit.
+
+**Parameters**
+
+- `adObj`: bid request object
+- `onCompleteListener`: listener object
+
+### removeUserKeyword
+
+Remove a key and all its associated values from a given Prebid Mobile ad unit.
+
+**Parameters**
+
+- `key`: String containing the key you want to remove.
+
+### removeUserKeywords
+
+Clear all key-value combinations from the Prebid Mobile ad unit.
+
+**Parameters**
+
+none
+
+### setAutoRefreshPeriodMillis
```

Comment:
Mention that we support only pbm refresh and do not adhere to the refresh of AdServer's. Suggest that they turnoff adobjects refresh